styles.css

```diff
@@ -81,8 +88,52 @@ header {
 	text-transform: uppercase;
 }
 
+main {
+	display: flex;
+	flex-flow: row nowrap;
```

Comment:
You might consider setting the width and allowing wrap, this would make the page easier to view on mobile.